### PR TITLE
♻️ Migrate Auth API reference guide

### PR DESCRIPTION
**Pull request by MeganSteenkamp**
Change to migrate API reference guide to new site. Currently using guides styling while awaiting UI input.

[📖 Migrate API Reference Guides](https://www.notion.so/centrapay/Migrate-API-Reference-Guides-13d91fa1f0a443c7b1d5c64aa353151c?pvs=4)

Test plan: Expect to see in dev site draft mode.

image

### CODE REVIEW

File: src/content/api/introduction.mdx [hunk 1]
```diff
@@ -1,8 +1,9 @@
 ---
-title: API Introduction
+title: Introduction
 description: Introduction to the API Reference
+draft: true
```
Comment (MeganSteenkamp) [inline src/content/api/introduction.mdx:4]: @AndyClifford was this meant to be on the existing page you made? I can't see where it was getting set 😅

File: src/content/api/auth.mdx [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+  order: 2
+---
+
+## Authenticating API Calls
+
+API calls can be authenticated by either providing an API key in the
+"X-Api-Key" header or by providing a user access token in the "Authorization"
+header.
+
+Org [Accounts](https://docs.centrapay.com/api/accounts) accessed with a user access token require the
```
Comment (MeganSteenkamp) [inline src/content/api/auth.mdx:16]: Are we doing absolute links while we do the migration?

File: src/content/api/auth.mdx [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+"X-Centrapay-Account" header to be provided. The "X-Centrapay-Account" header
+specifies the unique identifier of the Centrapay Org Account.
+
+### Authenticate with API key
+
+```bash [Request]
```
Comment (MeganSteenkamp) [inline src/content/api/auth.mdx:22]: This is quite ugly but will wait for design input 😅

File: src/content/api/auth.mdx (hunk 2 above)
Comment (Neeraj-Ghodla) [reply src/content/api/auth.mdx:16]: I think you'll have to use them until you get the file you are referencing into Astro. I suggest creating a ticket to change them to relative links when you are migrating.

**Review by Neeraj-Ghodla — APPROVED** [review]

**Pull request merged**